pycbc/waveform/bank.py

```diff
@@ -34,7 +34,12 @@
 from pycbc.pnutils import nearest_larger_binary_number
 import pycbc.io
 from copy import copy
+import numpy as np
+from math import *
```

Comment:
Don't do "from import *" in general. However, I don't see you using this import or the import of h5py. Please remove these.